sources/common/inc/LineCounter.h

```diff
@@ -95,13 +95,13 @@ namespace INSTRUMENT
             static const std::string HIGH_COUNTER_NAME;
 
             /// The counter name.
-            const std::string m_CounterName{};
+            const std::string m_CounterName {};
```

Comment:
Agreed. I'm trying not to but astyle is doing this and sometime I just miss to revert it. Maybe I will try to find a switch to fix it.